apps/condo/domains/common/utils/validation.utils.js

```diff
@@ -19,33 +19,48 @@ function hasDbFields (databaseRequired, resolvedData, existingItem, context, add
     }
     return hasAllFields
 }
-
-function hasRequestFields (requestFields = ['dv', 'sender'], resolvedData, context, addFieldValidationError) {
-    for (let field of requestFields) {
-        if (!resolvedData.hasOwnProperty(field)) {
-            if (context.req) {
-                const cookies = nextCookies({ req: context.req } )
-                if (cookies.hasOwnProperty(field)) {
-                    if (field === 'dv') {
-                        let parsed = parseInt(cookies[field])
-                        if (!isNaN(parsed)) {
-                            resolvedData[field] = parsed
-                            continue
-                        }
-                    }
-                    else
-                    { 
-                        resolvedData[field] = cookies[field]
-                        continue
-                    }
-                    
+function hasDvAndSenderFields (resolvedData, context, addFieldValidationError) {
+    let hasDvField = true
+    if (!resolvedData.hasOwnProperty('dv')) {
```

Comment:
I believe we would need a large confluence doc about the `dv` field in our system and how it works

- cookies,
- models

all that stuff